state/state.go

```diff
@@ -798,6 +803,10 @@ func (state *HelmState) flagsForUpgrade(helm helmexec.Interface, release *Releas
 		flags = append(flags, "--version", release.Version)
 	}
 
+	if release.Devel != nil && *release.Devel || state.HelmDefaults.Devel {
```

Comment:
I didn't notice it when I've written similar code for Verify below, but what happens when `helmDefaults.devel` is set to `true` but `releases[].devel` is explicitly set to `false` rather than omitted?

Should we write this `if release.Devel != nil && *release.Devel || ((release.Devel == nil || release.Devel != nil && !*release.Devel) && state.HelmDefaults.Devel)`?